authors/list_latex.py

```diff
@@ -7,7 +7,11 @@
 author_list = load(open("authors.json"))
 
 with open("../authors.tex", "wb") as f:
```

Comment:
Just so you know, your change actually did break it in Python 2 inside docker:
```
Traceback (most recent call last):
  File "./list_latex.py", line 13, in <module>
    f.write((u"%s%%\n" % author["name"]))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u0159' in position 3: ordinal not in range(128)
```
But it works with Python 2 on my desktop. The fix seems to require to add the following to `/usr/lib/python2.7/sitecustomize.py`:
```
import sys
reload(sys)
sys.setdefaultencoding('utf8')
```

So Python 2 unicode support seems seriously screwed up.